db/schema.rb

```diff
@@ -48,6 +48,7 @@
     t.string "name"
     t.string "photo"
     t.text "bio"
+    t.string "role", default: "admin"
```

Comment:
- [Required] I think it will be better to add the `role` column to the `users` table through a `migration` adding it directly to the `schema.rb` is not considered good practice since the column will be overwritten when the `migrations` are run using `db:migrate` command. Kindly create a migration that adds the `role` column to the database, this will ensure that the column is always created when the migrations are being created.